packages/toucan-js/src/sdk.ts

```diff
@@ -29,6 +30,7 @@ export class Toucan extends Scope {
               : [
                   requestDataIntegration(options.requestDataOptions),
                   linkedErrorsIntegration(),
+                  zodErrorsIntegration(),
```

Comment:
If this seems too heavy for a default, I can remove here and document as optional

Reply:
I think it's fine. Users can opt out if they want.